e2e/testdata/Docker_registry.def

```diff
@@ -17,7 +17,7 @@ from: registry:2.7.1
     dockerd --iptables=false --ip-forward=false --ip-masq=false --storage-driver=vfs &
     /.singularity.d/runscript &
     # wait until docker registry is up
-    while ! wget 127.0.0.1:5000 >/dev/null 2>&1; do sleep 0.2; done
+    while ! wget -O /dev/null 127.0.0.1:5000 >/dev/null 2>&1 ; do sleep 0.2; done
```

Comment:
Would it be better to use `wget`s silent flag over the redirect?

Reply:
```suggestion
    while ! wget -s -O /dev/null 127.0.0.1:5000 ; do sleep 0.2; done
```

Reply:
I double checked: wget does not have a silent flag, curl does.

Reply:
```suggestion
    while ! wget -q -O /dev/null 127.0.0.1:5000 ; do sleep 0.2; done
```